### DISCUSSION TITLE
Does GitHub Pages support SSL Cert other than Let's Encrypt?

### DISCUSSION THREAD

**Discussion by syeda-git** [community/community#128380] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, As a company, we would like to use a paid SSL certificate for our website. Is it possible? If yes, can you share instructions?

Is it possible for us to manage the certificate and not by GitHub?

Thanks...
Syed

**Comment by yoannchaudet** [comment]
Hello,

This is not something we support today.

**Comment by syeda-git** [comment]
@yoannchaudet We would like to use our own SSL certificate that we already have. We would like to upload it to GitHub. Is it possible?

**Reply by yoannchaudet** [reply]
It is not possible, sorry.

**Reply by syeda-git** [reply]
@yoannchaudet Thank you. 
Does Let's Encrypt validates only the subdomain that's in the cname record or does it validate the root domain? We want it to validate only  [opensource.hcl-software.com](http://opensource.hcl-software.com/) and not *.[hcl-software.com](http://hcl-software.com/). Basically we need confirmation that the cname and value are only for [opensource.hcl-software.com](http://opensource.hcl-software.com/)

**Reply by yoannchaudet** [reply]
I am not sure what you mean by "validate". If you setup `opensource.hcl-software.com` as your customer domain, assuming it is properly configured in your DNS, we will issue a certificate on your behalf, against Let's Encrypt for `opensource.hcl-software.com` (and potentially `www.opensource.hcl-software.com`). Nothing more, nothing less.
